sddi-base/Dockerfile

```diff
@@ -104,17 +104,17 @@ RUN set -ex && \
   pip install -U pip
 
 RUN set -ex && \
-  pip install -r https://raw.githubusercontent.com/ckan/ckanext-spatial/${CKANEXT_SPATIAL_VERSION}/requirements.txt && \
+  pip install -r https://raw.githubusercontent.com/MarijaKnezevic/ckanext-spatial/${CKANEXT_SPATIAL_VERSION}/requirements.txt && \
   curl -o /wheels/ckanext-spatial.txt \
-    https://raw.githubusercontent.com/ckan/ckanext-spatial/${CKANEXT_SPATIAL_VERSION}/requirements.txt && \
-  pip install -r https://raw.githubusercontent.com/ckan/ckanext-spatial/${CKANEXT_SPATIAL_VERSION}/requirements-postgis.txt && \
+    https://raw.githubusercontent.com/MarijaKnezevic/ckanext-spatial/${CKANEXT_SPATIAL_VERSION}/requirements.txt && \
+  pip install -r https://raw.githubusercontent.com/MarijaKnezevic/ckanext-spatial/${CKANEXT_SPATIAL_VERSION}/requirements-postgis.txt && \
   curl -o /wheels/ckanext-spatial-postgis.txt \
-    https://raw.githubusercontent.com/ckan/ckanext-spatial/${CKANEXT_SPATIAL_VERSION}/requirements-postgis.txt && \
+    https://raw.githubusercontent.com/MarijaKnezevic/ckanext-spatial/${CKANEXT_SPATIAL_VERSION}/requirements-postgis.txt && \
     ls -lah /wheels
 
 RUN set -ex && \
   pip wheel --wheel-dir=/wheels \
-    git+https://github.com/ckan/ckanext-spatial.git@${CKANEXT_SPATIAL_VERSION}#egg=ckanext-spatial
+    git+https://github.com/MarijaKnezevic/ckanext-spatial.git@${CKANEXT_SPATIAL_VERSION}#egg=ckanext-spatial
```

Comment:
Why do we need a fork of `ckan-spatial`? As discussed regarding the other currently forked ckan extensions, this is not desireable, as this means a new dependency, we have to maintain.

Reply:
@BWibo This for is a temporary solution for changes addopted for spatial extend in scheming. In the future the `ckanext-spatial` from comunity should be used.